android/build.gradle

```diff
@@ -33,16 +34,18 @@ android {
 repositories {
     mavenLocal()
     mavenCentral()
+    jcenter()
+    google()
 }
 
 dependencies {
-    annotationProcessor 'com.google.auto.value:auto-value:1.2'
-    implementation('org.tensorflow:tensorflow-lite:1.12.0')
+    annotationProcessor 'com.google.auto.value:auto-value:1.5.2'
+    implementation('org.tensorflow:tensorflow-lite:1.14.0')
     implementation('com.github.wendykierp:JTransforms:3.0')
-    implementation 'io.grpc:grpc-okhttp:1.16.1'
+    implementation 'io.grpc:grpc-okhttp:1.28.1'
     implementation 'com.google.code.gson:gson:2.8.5'
-    implementation 'com.google.cloud:google-cloud-speech:0.69.0-beta'
+    implementation 'com.google.cloud:google-cloud-speech:1.20.0' // NB 1.21+ fails at either build or runtime.
```

Comment:
That's...interesting. Think we're using 1.21 in Android demos.

Reply:
posting here for the public record: 
```
behavior: app loads, opens, and then immediately closes

stack trace (adb logcat *:S ReactNative:V ReactNativeJS:V RNSpokestack:V AndroidRuntime:E):

04-20 10:41:21.294 29272 29323 E AndroidRuntime: FATAL EXCEPTION: create_react_context
04-20 10:41:21.294 29272 29323 E AndroidRuntime: Process: com.lesnar, PID: 29272
04-20 10:41:21.294 29272 29323 E AndroidRuntime: java.lang.UnsatisfiedLinkError: dlopen failed: library "libconscrypt_openjdk_jni-linux-x86.so" not found
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at java.lang.Runtime.loadLibrary0(Runtime.java:1087)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at java.lang.Runtime.loadLibrary0(Runtime.java:1008)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at java.lang.System.loadLibrary(System.java:1661)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at org.conscrypt.NativeLibraryUtil.loadLibrary(NativeLibraryUtil.java:54)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at java.lang.reflect.Method.invoke(Native Method)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at org.conscrypt.NativeLibraryLoader$1.run(NativeLibraryLoader.java:297)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at org.conscrypt.NativeLibraryLoader$1.run(NativeLibraryLoader.java:289)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at java.security.AccessController.doPrivileged(AccessController.java:43)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at org.conscrypt.NativeLibraryLoader.loadLibraryFromHelperClassloader(NativeLibraryLoader.java:289)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at org.conscrypt.NativeLibraryLoader.loadLibrary(NativeLibraryLoader.java:262)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at org.conscrypt.NativeLibraryLoader.load(NativeLibraryLoader.java:162)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at org.conscrypt.NativeLibraryLoader.loadFirstAvailable(NativeLibraryLoader.java:106)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at org.conscrypt.NativeCryptoJni.init(NativeCryptoJni.java:50)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at org.conscrypt.NativeCrypto.<clinit>(NativeCrypto.java:63)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at org.conscrypt.NativeCrypto.checkAvailability(NativeCrypto.java:80)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at org.conscrypt.Conscrypt.checkAvailability(Conscrypt.java:111)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at org.conscrypt.Conscrypt.isAvailable(Conscrypt.java:50)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at okhttp3.internal.platform.ConscryptPlatform.<clinit>(ConscryptPlatform.kt:111)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at okhttp3.internal.platform.android.ConscryptSocketAdapter.buildIfSupported(ConscryptSocketAdapter.kt:64)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at okhttp3.internal.platform.AndroidPlatform.<init>(AndroidPlatform.kt:45)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at okhttp3.internal.platform.AndroidPlatform$Companion.buildIfSupported(AndroidPlatform.kt:239)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at okhttp3.internal.platform.Platform$Companion.findPlatform(Platform.kt:211)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at okhttp3.internal.platform.Platform$Companion.access$findPlatform(Platform.kt:179)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at okhttp3.internal.platform.Platform.<clinit>(Platform.kt:180)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at okhttp3.OkHttpClient.<init>(OkHttpClient.kt:219)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at okhttp3.OkHttpClient$Builder.build(OkHttpClient.kt:955)
04-20 10:41:21.294 29272 29323 E AndroidRuntime:    at com.facebook.react.devsupport.DevServerHelper.<init>(DevServerHelper.java:132)

Tracked the offending cause down to two lines in react-native-spokestack/android/build.gradle:

-    implementation 'io.grpc:grpc-okhttp:1.16.1'
+    implementation 'io.grpc:grpc-okhttp:1.25.0'
-    implementation 'com.google.cloud:google-cloud-speech:0.69.0-beta'
+    implementation 'com.google.cloud:google-cloud-speech:1.22.2'
```